packages/iTwinUI-react/src/core/Table/Table.tsx

```diff
@@ -813,7 +813,15 @@ export const Table = <
                               />
                             )}
                             {showSortButton(column) && (
-                              <div className='iui-table-cell-end-icon'>
+                              <div
```

Comment:
In CSS it is the whole cell that has `tabIndex`

Reply:
That is true. I feel as though it's confusing to tab onto the filter button for filtering and tab onto the header cell for sorting.

Reply:
But this is how even mouse clicks work - you press on the whole cell not the icon.

Reply:
Yep, you're totally right. Do you think the header cell be tab-focusable if there's no sort or filter button? The CSS demo shows the header cell being tab-focusable when there's the filter button but not the sort button, but doesn't have an example of a regular/text header cell with no filter or sort buttons.

Reply:
I followed how the mouse clicks/hovers work for the react table. The header cells change color on hover, so I made it so that all header cells can be tab-focusable.